portal-impl/src/content/Language_fr_CA.properties

```diff
@@ -5214,7 +5214,7 @@ the-selected-user-group-is-associated-with-deactivated-users.-do-you-want-to-pro
 the-selected-web-content-no-longer-exists=Le contenu de Web choisi n'existe plus.
 the-setup-could-not-be-found=L'installation n'a pas pu être trouvée.
 the-sign-in-process-is-temporarily-disabled-because-the-portal-is-undergoing-a-routine-maintenance-upgrade=Le processus d'authentification est temporairement désactivé parce que le portail est en cours de maintenance. Le processus d'authentification sera réactivé une fois que la mise-à-jour sera complétée. Merci de votre patience.
-the-signature-below-will-be-added-to-each-outgoing-message=La signature ci-dessous sera ajoutée à chaque message envoyé.
+the-signature-below-will-be-added-to-each-outgoing-message=La signature ci-dessous sera ajoutée à chaque message envoyé.
```

Comment:
Unsure if `0xa0` (non-breaking space) is critical here, to avoid breaking lines.